### ISSUE TITLE
Custom Ratios?

### ISSUE THREAD

**Issue by Arroyodorado** [lessthanzero/Formr#3]
How about adding a menu option for entering custom ratios, eg. 5:3, 16:10 and so on?

**Comment by lessthanzero** [comment]
@Arroyodorado right now you can add custom ratios with a `:` key in layer name

**Comment by Arroyodorado** [comment]
Yes, I know. I just thought a dialog would be more convenient... 8-)  And I wouldn't need to mess with my layer names.
